components/markdown/components/code-block.js

```diff
@@ -36,9 +42,33 @@ class CodeBlock extends React.PureComponent {
 
   render() {
     const { children, language } = this.props;
+
+    // Add Styling to button here.
+    // TODO decide if the button should be left or right aligned.
+    const CopyButton = () => (
+      <div
+        style={{
+          textAlign: 'right',
+          marginBottom: '-32px',
+        }}
```

Comment:
Would prefer absolute positioning, so that the position will be more resilient against size changes.

You could do something like `position: 'absolute', top: 8, right: 8`, but note that this will require the parent to have set  `position: 'relative'` explicitly.

Reply:
Hi , I have made the changes that you have suggested.